iOS/DuckDuckGo/SettingsViewModel.swift

```diff
@@ -102,6 +103,10 @@ final class SettingsViewModel: ObservableObject {
         case networkProtection
     }
 
+    var isPaidAIChatEnabled: Bool {
```

Comment:
The meaning here is obvious to us but maybe we can add a one-line doc comment to explain this property, e.g. - "Indicates if the Paid AI Chat feature flag is enabled for the current user/session."